exonum/src/blockchain/schema.rs

```diff
@@ -51,24 +51,27 @@ define_names!(
 encoding_struct! (
     /// Configuration index.
     struct ConfigReference {
-        /// The height, starting from which this configuration becomes actual.
+        /// Height since which this configuration becomes actual.
         actual_from: Height,
         /// Hash of the configuration contents that serialized as raw bytes vec.
         cfg_hash: &Hash,
     }
 );
 
 encoding_struct! (
-    /// Transaction location in block.
+    /// Transaction location in a block.
+    /// The given entity defines the block where the transaction was
+    /// included and the position of this transaction in that block.
     struct TxLocation {
-        /// Height of block in the blockchain.
+        /// Height of the block where the transaction was included.
         block_height: Height,
-        /// Index in block.
+        /// Number designated to this transaction in the block.
```

Comment:
I find the phrasing "Number designated to this transaction" overly obscure; IMO, something like "zero-based index" (or "zero-based position") would convey the concept better.